src/view/screens/Home.tsx

```diff
@@ -218,26 +227,39 @@ function HomeScreenReady({
       onPageSelected={onPageSelected}
       onPageScrollStateChanged={onPageScrollStateChanged}
       renderTabBar={renderTabBar}>
-      <FeedPage
-        key={homeFeed}
-        testID="followingFeedPage"
-        isPageFocused={selectedFeed === homeFeed}
-        feed={homeFeed}
-        feedParams={homeFeedParams}
-        renderEmptyState={renderFollowingEmptyState}
-        renderEndOfFeed={FollowingEndOfFeed}
-      />
-      {customFeeds.map(feed => {
-        return (
-          <FeedPage
-            key={feed}
-            testID="customFeedPage"
-            isPageFocused={selectedFeed === feed}
-            feed={feed}
-            renderEmptyState={renderCustomFeedEmptyState}
-          />
-        )
-      })}
+      {allFeeds.length ? (
+        allFeeds.map(feed => {
+          if (feed === 'following') {
+            return (
+              <FeedPage
+                key={feed}
+                testID="followingFeedPage"
+                isPageFocused={selectedFeed === feed}
+                feed={feed}
+                feedParams={homeFeedParams}
+                renderEmptyState={renderFollowingEmptyState}
+                renderEndOfFeed={FollowingEndOfFeed}
+              />
+            )
+          }
+          const savedFeedConfig = preferences.savedFeeds.find(f => {
+            return feed.includes(f.value)
```

Comment:
Feels off that we have to do a loop within a loop here. Not the end of the world but I think this means iterating over `allFeeds` might need some restructuring (should we just iterate over `preferences.savedFeeds` instead?)

Reply:
Oh that's a good shout, it's 1:1 now, we might as well loop over `prefs.savedFeeds.filter(f => f.pinned)` and create the `FeedDescriptor`s in situ

Reply:
found a neat way to score this